### PR TITLE
.NET Interactive: Add_try_dotnet_to_batch_26a

### PR DESCRIPTION
**Pull request by v-mepa**
fyi @mairaw 

## Add_try_dotnet_to_batch_26a

URL | Pageviews | BatchNo. | Comments
-- | -- | -- | --
https://docs.microsoft.com/en-us/dotnet/api/system.datetime.dayofyear | 1182 | Batch 26 |  
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.equalitycomparer-1.default | 1176 | Batch 26 |  
https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.thenby | 1167 | Batch 26 |  
https://docs.microsoft.com/en-us/dotnet/api/system.type.getconstructor | 1120 | Batch 26 | Duplicate samples for all   overloads? Potentially move it to the MemberGroup.
https://docs.microsoft.com/en-us/dotnet/api/system.linq.queryable.cast | 1105 | Batch 26 |  
https://docs.microsoft.com/en-us/dotnet/api/system.type.issubclassof | 1100 | Batch 26 |  
https://docs.microsoft.com/en-us/dotnet/api/system.text.stringbuilder.remove | 1085 | Batch 26 |  
https://docs.microsoft.com/en-us/dotnet/api/system.buffer | 1053 | Batch 26 |  
https://docs.microsoft.com/en-us/dotnet/api/system.multicastdelegate | 1035 | Batch 26 |  
https://docs.microsoft.com/en-us/dotnet/api/system.delegate.createdelegate | 1023 | Batch 26 | Except for the samples with   the following intro: The code example produces no output.
https://docs.microsoft.com/en-us/dotnet/api/system.decimal.-ctor | 1021 | Batch 26 | Fix sample output for sample   before the last
https://docs.microsoft.com/en-us/dotnet/api/system.converter-2 | 985 | Batch 26 |

### CODE REVIEW

File: xml/System/Type.xml [hunk 1]
```diff
@@ -2100,16 +2103,7 @@ The `filter` argument can be a custom delegate of type <xref:System.Reflection.M
 >  You cannot omit parameters when looking up constructors and methods. You can only omit parameters when invoking.  
   
  If the current <xref:System.Type> represents a constructed generic type, this method returns the <xref:System.Reflection.ConstructorInfo> with the type parameters replaced by the appropriate type arguments. If the current <xref:System.Type> represents a type parameter in the definition of a generic type or generic method, this method always returns `null`.  
-  
-   
-  
-## Examples  
- The following program obtains the type of `MyClass1` class, gets the <xref:System.Reflection.ConstructorInfo> object matching the specified binding flags, and displays the signature of the constructor.  
-  
- [!code-cpp[Type_GetConstructor2#1](~/samples/snippets/cpp/VS_Snippets_CLR/Type_GetConstructor2/CPP/type_getconstructor2.cpp#1)]
```
Comment (mairaw) [inline xml/System/Type.xml:2109]: I noticed that for this page the samples were not the same, so we should keep it in the member itself and simply convert the original format to the new syntax instead.

**Review by mairaw — CHANGES_REQUESTED** [review]
Changes look good. I've made one change to fix the build warning already but there's another larger comment that also needs to be addressed. Don't forget to do git pull before making the new changes.

File: xml/System/Type.xml (hunk 1 above)
Comment (v-mepa) [reply xml/System/Type.xml:2109]: Does that mean removing the new syntax from the member group and adding them back in member with new syntax.

Comment (mairaw) [reply xml/System/Type.xml:2109]: Correct.

File: xml/System/Type.xml [hunk 2]
```diff
@@ -1912,7 +1912,7 @@ The `filter` argument can be a custom delegate of type <xref:System.Reflection.M
       </AssemblyInfo>
       <Docs>
         <summary>Gets a specific constructor of the current <see cref="T:System.Type" />.</summary>
-      </Docs>
+             </Docs>
```
Comment (mairaw) [inline xml/System/Type.xml:1915]:
```suggestion
      </Docs>
```

**Review by mairaw — APPROVED** [review]

**Pull request merged**